hermeznetwork/hermez-node
# set eth_tx_hash by default to 0x0

### What does this PR does?

This PR modifies the migration 3 and set the default value of the eth_tx_hash column to 0x0.

### How to test?

Run the migration and check the default value ofeth_tx_hash with the next query:
```
SELECT column_name, column_default
FROM information_schema.columns
WHERE (table_schema, table_name) = ('public', 'batch')
ORDER BY ordinal_position;
```

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
